cognite/powerops/custom_modules/power_model_v1/data_models/containers/ShopModelToAssets.container.yaml

```diff
@@ -0,0 +1,34 @@
+space: "{{power_ops_models}}"
+externalId: ShopModelToAssets
+name: ShopModelToAssets
```

Comment:
**nitpick:** I'd maybe rename this to `ShopModelWithAssets`? I think that is a more accurate name from my understanding of the concept.

Reply:
It's a type for connecting a Shop model to PowerAssets, hence the name, but it has become a bit broader, and maybe ShopModelWithAssets is just as good a name...

Reply:
The English grammar in my head (probably mostly the programmer in me 🤣) interprets "ModelToAssets" as some sort of generating object since this is a node and not an edge. Hence why, "ModelWithAssets" makes more sense to me if it's going to be a node where the model and its related assets are already grouped together on the node.

**note:** It could be that this concept would be better modeled as an _edge_ instead of a _node_ (and then ModelToAsset would work semantically in my head 😜) so maybe we should consider that